packages/react-native-reanimated/src/mutables.ts

```diff
@@ -219,19 +220,20 @@ function experimental_makeMutableNative<Value>(initial: Value): Mutable<Value> {
     get value(): Value {
       checkInvalidReadDuringRender();
       if (dirtyFlag.getBlocking()) {
-        const uiValueGetter = executeOnUIRuntimeSync((sv: Mutable<Value>) => {
-          sv.setDirty!(false);
-          return sv.value;
-        });
+        const uiValueGetter = (svArg: Mutable<Value>) =>
+          runOnUISync((sv) => {
+            sv.setDirty!(false);
+            return sv.value;
+          }, svArg);
```

Comment:
```suggestion
        const uiValueGetter = runOnUISync.bind(null, (sv: Mutable<Value>) => {
          sv.setDirty!(false);
          return sv.value;
        });
```

Not sure if TS will be satisfied, but maybe try using `.bind` to reduce nesting?